test/pkg/integration/workspace.go

```diff
@@ -127,14 +128,20 @@ func LaunchWorkspaceDirectly(it *Test, opts ...LaunchWorkspaceDirectlyOpt) (res
 			return
 		}
 	}
+	if workspaceImage == "" {
+		it.t.Fatalf("cannot start workspaces without a workspace image (required by resolver)")
+	}
 
 	ideImage := options.IdeImage
 	if ideImage == "" {
-		cfg, err := it.GetServerConfig()
+		cfg, err := it.GetServerIDEConfig()
 		if err != nil {
-			it.t.Fatalf("cannot find server config: %q", err)
+			it.t.Fatalf("cannot find server IDE config: %q", err)
+		}
+		ideImage = cfg.IDEImageAliases.Code
+		if ideImage == "" {
+			it.t.Fatalf("cannot start workspaces without an IDE image (required by resolver)")
```

Comment:
which resolver does this refer to?

Reply:
registry-facade - error message updated to reflect